changelogs/22.04

```diff
@@ -0,0 +1,52 @@
+* Pop!_OS is now based on Ubuntu 22.04.
+
+* COSMIC
+** GNOME 42 support has been added for COSMIC.
+** The Shell Theme for GNOME 42 has been updated.
+** Icons are now SVG-based instead of PNG-based.
+** Dark and light modes can have independent backgrounds.
+** Settings for Desktop Options, Background, Appearance, Dock, and Workspaces are now searchable in Launcher.
+** The Pop robot face has been added as the default avatar.
+** Improvements to cosmic-workspaces:
+    ** An issue with the location of the Workspace picker on secondary displays has been fixed.
+    ** An issue with the scaling of the Workspace picker on displays of varying resolutions has been fixed.
+    ** Performance has been improved.
+    ** An issue with alignment on HiDPI displays has been fixed.
+    ** The OSD picker is now smaller.
+** COSMIC logging has been limited, resulting in increased system performance.
+
+* Pop!_OS System Updater
+** New upgrade and options for installing upgrades are now available in Settings > OS Upgrade and Recovery.
```

Comment:
I didn't get what "New upgrade and upgrade installation options" meant the first time, but I see with this rewording that the entire point is referring to the auto update feature. I would say "options for installing updates" since we generally use "upgrade" to refer to release upgrades and "update" to refer to installing updates on the current release. You can probably just say "New options for installing updates" and not have the word "upgrades" in this line at all (I still don't understand why the word "upgrade" is being used twice in this bullet point as-is.)